pages/documentation/actions/bg-function.md

```diff
@@ -0,0 +1,18 @@
+---
+title: bg-function(deprecated)
```

Comment:
Currently titles are missing from the content pages. I think we can add them in `default.ejs`.